sqlite/migrator.go

```diff
@@ -132,3 +211,27 @@ func dropExtension(filename string) string {
 
 	return filename[:idx]
 }
+
+// filteredMigrations filters the list of migration files to include only "up" migrations or only "down" migrations. The
+// returned list is sorted in ascending order.
+func filteredMigrations(list []fs.DirEntry, filterForUps bool) []fs.DirEntry {
```

Comment:
Could we instead rename the existing "up" migrations to be` .up.sql` and have separate embeddings for down and up?

Reply:
Done! Nice that this eliminates the need for the filtering logic.